docs/guide/testrunner/gettingstarted.md

```diff
@@ -32,6 +32,7 @@ Options:
   --watch               watch specs for changes
   --logLevel, -l        level of logging verbosity (default: silent)
   --coloredLogs, -c     if true enables colors for log output (default: true)
+  --deprecationWarnings if true use of deprecated commands will output a warning (default: true)
```

Comment:
if you want to add it here you need to add support for it in `cli.js`. If you ask me .. not needed